website/static/js/pages/project-base-page.js

```diff
@@ -32,4 +32,40 @@ $(document).ready(function() {
     $.getJSON(node.urls.api, function(data) {    
         $('body').trigger('nodeLoad', data);
     });
+
+    var target = 50, smallScreenSize = 767,
+    timeout = null;
+    var checkPanelPosition = function() {
+        var bodyWidth = $(document.body).width();
+        var scrollTopPosition = $(window).scrollTop();
+        if (bodyWidth <= smallScreenSize) {
+            if (scrollTopPosition >= target) {
+                $('.cp-handle').css('margin-top', 50);
+            }
+            else {
+                $('.cp-handle').css('margin-top', 95 - scrollTopPosition);
+            }
+        }
+    };
+
+    checkPanelPosition(); /* Init when refreshing the page*/
+    $(window).scroll(function () {
+        if (!timeout) {
+            timeout = setTimeout(function () {
+                clearTimeout(timeout);
+                timeout = null;
+                checkPanelPosition();
+            }, 10);
+        }
+    });
+
+    $( window ).resize(function() {
+        var bodyWidth = $(document.body).width();
+        var scrollTopPosition = $(window).scrollTop();
+        if (bodyWidth > smallScreenSize || scrollTopPosition < target) {
+            $('.cp-handle').css('margin-top', 95);
```

Comment:
@sloria Make sense! It is the margin for comment panel icon. When project nav bar is hidden, the margin should be 50 and when it is shown, margin should be 95.